src/test/java/api/items/ItemApiExamples.java

```diff
@@ -937,6 +957,158 @@ public void canRemoveBarcodeFromAnExistingItem()
+      is(source.getJsonObject(PERSONAL_KEY).getString(LAST_NAME_KEY)));
+    assertThat(source.getJsonObject(PERSONAL_KEY).getString(FIRST_NAME_KEY),
+      is(source.getJsonObject(PERSONAL_KEY).getString(FIRST_NAME_KEY)));
+
+    selfLinkRespectsWayResourceWasReached(createdItem);
+    selfLinkShouldBeReachable(createdItem);
+  }
+
+  @Test
+  public void canUpdateAnItemWithExistingCirculationNote()
+    throws InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+
+    JsonObject createdInstance = createInstance(
+      smallAngryPlanet(UUID.randomUUID()));
+
+    UUID holdingId = holdingsStorageClient.create(
+      new HoldingRequestBuilder()
+        .forInstance(UUID.fromString(createdInstance.getString("id"))))
+      .getId();
+
+    UUID itemId = UUID.randomUUID();
+
+    JsonObject newItemRequest = new ItemRequestBuilder()
+      .withId(itemId)
+      .forHolding(holdingId)
+      .withBarcode("645398607547")
+      .canCirculate()
+      .temporarilyInReadingRoom()
+      .withCheckInNote()
+      .create();
+
+    DateTime createItemRequestMade = DateTime.now();
+
+    itemsClient.create(newItemRequest);
+
+    String createdItemCirculationNoteDate = itemsClient.getById(itemId)
+      .getJson()
+      .getJsonArray(CIRCULATION_NOTES_KEY)
+      .getJsonObject(0)
+      .getString(DATE_KEY);
+
+    JsonObject updateItemRequest = newItemRequest.copy()
+      .put("status", new JsonObject().put("name", "Checked Out"));
+
+    itemsClient.replace(itemId, updateItemRequest);
+
+    Response getResponse = itemsClient.getById(itemId);
+
+    assertThat(getResponse.getStatusCode(), is(200));
+    JsonObject updatedItem = getResponse.getJson();
+
+    assertThat(updatedItem.containsKey("id"), is(true));
+    assertThat(updatedItem.getString("title"), is("Long Way to a Small Angry Planet"));
+    assertThat(updatedItem.getString("barcode"), is("645398607547"));
+    assertThat(updatedItem.getJsonObject("status").getString("name"), is("Checked Out"));
+
+    JsonObject materialType = updatedItem.getJsonObject("materialType");
+
+    assertThat(materialType.getString("id"), is(ApiTestSuite.getBookMaterialType()));
+    assertThat(materialType.getString("name"), is("Book"));
+
+    JsonObject permanentLoanType = updatedItem.getJsonObject("permanentLoanType");
+
+    assertThat(permanentLoanType.getString("id"), is(ApiTestSuite.getCanCirculateLoanType()));
+    assertThat(permanentLoanType.getString("name"), is("Can Circulate"));
+
+    assertThat("Item should not have permanent location",
+      updatedItem.containsKey("permanentLocation"), is(false));
+
+    assertThat(updatedItem.getJsonObject("temporaryLocation").getString("name"), is("Reading Room"));
+
+    JsonObject checkInNote = updatedItem.getJsonArray(CIRCULATION_NOTES_KEY).getJsonObject(0);
+
+    assertThat(checkInNote.getString(DATE_KEY), not(createdItemCirculationNoteDate));
```

Comment:
Does this check that the date the note was added does not change on update?

Reply:
The check reflects that the date is changed when the note is updated.